python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,138 @@
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnGeneralServiceException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class {{ operation }}Handler extends BaseHandlerStd {
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<SdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        return ProgressEvent.progress(model, callbackContext)
+
+            // STEP 1 [check if resource already exists]
+            // if target API does not support 'ResourceAlreadyExistsException' then following check is required
+            //.then(progress -> checkForPreCreateResourceExistence(proxy, request, progress))
+
+            // STEP 2 [create/stabilize progress chain - required for resource creation]
+            .then(progress ->
+                // If your service API throws 'ResourceAlreadyExistsException' for create requests then CreateHandler can return just proxy.initiate construction
+                // STEP 2.0 [initialize a proxy context]
+                proxy.initiate("{{ call_graph }}::{{ operation }}", proxyClient, model, callbackContext)
+
+                    // STEP 2.1 [TODO: construct a body of a request]
+                    .request(Translator::translateToCreateRequest)
+
+                    // STEP 2.2 [TODO: make an api call]
+                    .call(this::createResource)
+
+                    // STEP 2.3 [TODO: stabilize is describing the resource until it is in a certain status]
+                    .stabilize(this::stabilizedOnCreate)
+                    .progress())
+
+            // STEP 3 [TODO: describe call/chain to return the resource model]
+            .then(progress -> new ReadHandler().handleRequest(proxy, request, callbackContext, proxyClient, logger));
+    }
+
+    /**
+     * If your service API does not distinguish duplicate create requests against some identifier (e.g; resource Name)
+     * and instead returns a 200 even though a resource already exists, you must first check if the resource exists here
+     * NOTE: If your service API throws 'ResourceAlreadyExistsException' for create requests this method is not necessary
```

Comment:
i would include the word "idempotent" in here somewhere. 

"If your service API is not idempotent, meaning it does not distinguish..." or something like that